src/chart/Sankey.tsx

```diff
@@ -667,6 +669,7 @@ export class Sankey extends PureComponent<Props, State> {
       <div
         className={classNames('recharts-wrapper', className)}
         style={{ ...style, position: 'relative', cursor: 'default', width, height }}
+        role="region"
```

Comment:
Why apply this role here? https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Roles/Region_role